charts/nri-bundle/requirements.yaml

```diff
@@ -1,9 +1,26 @@
 dependencies:
+  # Old newrelic-infrastructure chart, now deprecated.
   - name: newrelic-infrastructure
     repository: file://../newrelic-infrastructure
     condition: infrastructure.enabled
     version: 2.7.3
 
+  # New nri-kubernetes chart with compatibility layer.
+  - name: nri-kubernetes
+    alias: newrelic-infrastructure
+    repository: file://../nri-kubernetes
+    # Notes of why this is overengineered:
+    #  * This chart is still using Helm 2 so we cannot set various conditions like `infrastructure.enabled,newrelic-infrastructure.compatibilityLayer`
+    #  * Helm 2 also does not support `import-values` so we have to alias the new chart to the old one to get the values from it.
+    condition: newrelic-infrastructure.compatibilityLayer
```

Comment:
If we don't set this to `false` (or `true`) in the values, helm will ignore the condition and render the chart.

Reply:
Really good point. I would have to think about it.